src/ui/map.js

```diff
@@ -362,6 +362,8 @@ class Map extends Camera {
     _averageElevationLastSampledAt: number;
     _averageElevation: EasedVariable;
     _runtimeProjection: ProjectionSpecification | void | null;
+    _containerWidth: number;
+    _containerHeight: number
```

Comment:
can we a add a private getter to Map to access these, something like 
```
_getCurrentContainerDimensions(): [number, number] {
 return { witdth: this._containerWidth, height: this._containerHeight};
}
```

just to prevent accidentally writing to these values from outside of Map